draft-ietf-mops-streaming-opcons.md

```diff
@@ -312,7 +315,7 @@ here, but it seems worth making the point that demand projections can
 be used to help with e.g. power consumption with routing architectures
 that provide for modular scalability.
 
-##Unpredictable Usage Profiles {#sec-unpredictable}
+##Unpredictable Usage Profiles {#sec-unpredict}
```

Comment:
Is there a label length limit? Just curious.

Reply:
@squarooticus - not that I haven encountered. I usually use shorter labels, and couldn't believe I used one that long - since it was only used in one place so far (and I typed that part, too), I shortened it, but that was only my preference.